### DISCUSSION TITLE
Losing syntax highlighting in JS template literal when cursor is over the text

### DISCUSSION THREAD

**Discussion by olalonde** [nvim-treesitter/nvim-treesitter#4445] [Q&A]
When my cursor goes over the template literal, I lose all syntax highlighting and it just becomes yellow. I was wondering if there was a way to stop that from happening.

https://user-images.githubusercontent.com/416585/222932223-afeef807-1da0-4af6-9e12-f770582fb99e.mov

**Comment by amaanq** [comment]
Could be a theme issue. Check the outputs of `:lua vim.show_pos()` inside and right outside the node

**Reply by theHamsta** [reply]
I can't see the video any more , but in many similar cases the internal query match limit was hit and which causes some matches to flicker on/off depending on cursor position. I can't check on mobile right now whether the match limit is configurable right now.

**Reply by theHamsta** [reply]
Can now see the video. This seems to be a different issue.
